tests/components/laundrify/test_sensor.py

```diff
@@ -0,0 +1,106 @@
+"""Test the laundrify sensor platform."""
+
+from datetime import timedelta
+import json
+import logging
+from unittest.mock import AsyncMock, patch
+
+from laundrify_aio import LaundrifyDevice
+import pytest
+
+from homeassistant.components.laundrify.const import DOMAIN, MANUFACTURER, MODELS
+from homeassistant.components.laundrify.sensor import LaundrifyPowerSensor
+from homeassistant.components.sensor import SensorDeviceClass
+from homeassistant.const import (
+    ATTR_DEVICE_CLASS,
+    ATTR_UNIT_OF_MEASUREMENT,
+    STATE_UNKNOWN,
+    UnitOfPower,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import device_registry as dr
+from homeassistant.util import slugify
+from homeassistant.util.dt import utcnow
+
+from . import setup_config_entry
+
+from tests.common import async_fire_time_changed, load_fixture
+
+# Add logging to trace the flow
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
This seems an odd place to do this, this affects all tests.

```suggestion
```

Reply:
Addressed in https://github.com/home-assistant/core/pull/121378/commits/5efc1eef0367b3c61c5b7d73d29253e874b897c5